biomejs/biome
# doc(ja): translate `Formatter` doc into Japanese

## Summary
Partial work of #880. ` formatter/index.mdx`




## Test Plan
- Check if all links works as expecetd
- Check if all translations make sense
